tests/test_dynamodb/exceptions/test_dynamodb_exceptions.py

```diff
@@ -487,8 +538,8 @@ def test_query_unused_attribute_name(table_name=None):
 
     with pytest.raises(ClientError) as exc:
         table.query(
-            KeyConditionExpression="(#0 = x) AND (begins_with(#1, a))",
-            ExpressionAttributeNames={"#0": "pk", "#1": "sk", "#count": "count"},
+            KeyConditionExpression="(#0 = x)",
```

Comment:
There were two things wrong with this test - 'sk' cannot be used, and 'count' is unused. Moto validates both, and the validation order is wrong. This is not particularly important IMO - I've just removed the 'sk' attr to make sure it only chokes on the unused 'count' attribute, as that is the only thing we're trying to test here.